src/sec_edgar_bulker/core/proxy_manager.py

```
@@ -0,0 +1,256 @@
+"""Proxy manager for SEC EDGAR requests."""
+
+import random
+from typing import List, Dict, Optional, Tuple, Set
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)